x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_container.tsx

```diff
@@ -76,8 +85,6 @@ const LatestFindingsByResource = ({ dataView }: { dataView: DataView }) => {
         setQuery={(query) => {
           setUrlQuery({ ...query, pageIndex: 0 });
         }}
-        query={urlQuery.query}
-        filters={urlQuery.filters}
```

Comment:
this was removed because (as it turns out) passing the filters/query we get from the search bar back to it doesn't display it properly (and makes filters not editable) 

it also seems other usages of `SearchBar` also does not include these props

so instead of being controlled via these props, we use (like others) `filterManager.setAppFilters` and `queryString.setQuery` to set the values we need. 

as a side note, it's worth pointing out we will also need to account for global kibana app filters to pick up user state across apps.